sherpa.sh

```diff
@@ -1,14 +1,18 @@
 package: SHERPA
 version: "%(tag_basename)s"
-tag: "v2.2.4-alice1"
-source: https://github.com/alisw/SHERPA
+tag: "v2.2.8-alice1"
+source: https://github.com/mfasDa/SHERPA
```

Comment:
The intent is likely to change this l.4 before the actual merging, to pick-up the tag v2.2.8-alice1
from https://github.com/alisw/SHERPA and not from Markus' fork used for development
See the parallel https://github.com/alisw/SHERPA/pull/2 where we are about to create new branch v2.2.8 and then tag v2.2.8-alice1 for alisw/SHERPA

Reply:
Of course, my own fork was just a temporary solution for the testing phase.